bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -254,7 +254,7 @@ func CreateKfDefFromOptions(options map[string]interface{}) (*kfdefsv3.KfDef, er
 	}
 
 	// If a config file is specified, construct the KfDef entirely from that.
-	configFile := options[string(kftypesv3.CONFIG)].(string)
+	configFile := options[string(kftypesv3.FILE)].(string)
```

Comment:
Can we just delete the function CreateKfDefFromOptions? I don't think it is used anymore.
This function was used for backwards compatibility in 0.6 to specify all the different command line flags and then create a KfDef from it.

Reply:
I deleted all functions that relies on options.